File: new/dart-maor/dialer-is-done.dart
// ⚛️ אטום-Dart (דרגת-חוזה) · isDone — האם התור ריק.
// מוצא: maor-system/src/lib/dialer.ts:97 (isDone) · המקור: new/atoms/dialer-is-done.mjs.
// טוהר: פונקציית top-level עצמאית, אפס import (dart:core בלבד). חוק-4 — זהה-ביט למקור-JS.
//  • `c.queue.length === 0` ⇒ queue.isEmpty.

/// True when the queue is empty. Verbatim port of new/atoms/dialer-is-done.mjs.
bool isDone(Map<String, dynamic> c) {
  return (c['queue'] as List).isEmpty;
}

File: new/atoms/dialer-is-done.mjs
/** חוט · dialer-is-done — Golden. חוזה: dialer-is-done.contract.md
 * מוצא: dialer.ts:97 (isDone). חוק-4 verbatim. האם התור ריק. טהור.
 */
export function isDone(c) {
  return c.queue.length === 0;
}
